elements/pfe-health-index/src/pfe-health-index.scss

```diff
@@ -1,23 +1,26 @@
 @import "../../pfe-sass/pfe-sass";
-// @include pfe-theme-contexts; // imports on="light" etc support
-
-/// ===========================================================================
-/// Component Specific SASS Vars
-/// ===========================================================================
-
-$pfe-health--width: 10px;
-$pfe-health--height: $pfe-health--width * 2;
-$pfe-health-mini--width: 22px;
-$pfe-health-mini--height: $pfe-health-mini--width;
-
-$pfe-health--color-bg: #fff;
-
-$pfe-health--color-a: #3f9c35;
-$pfe-health--color-b: #92d400;
-$pfe-health--color-c: #efaa00;
-$pfe-health--color-d: #ec7a08;
-$pfe-health--color-e: #cc0000;
-$pfe-health--color-f: #a30000;
+@include pfe-theme-contexts; // imports on="light" etc support
+
+$LOCAL: health-index;
+
+$colors: (
+  a: #3f9c35,
+  b: #92d400,
+  c: #efaa00,
+  d: #ec7a08,
+  e: #cc0000,
+  f: #523333
+);
+
+// TODO: Migrate to theme values?
```

Comment:
any reason not to do this?

Reply:
Right now, those colors don't exist in the theme.  I think it would need some discovery before doing that work.

Reply:
This is np (totally not needed in this PR) but I was thinking the [feedback colors](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-sass/maps/_colors.scss#L134) would do the trick.

Reply:
You can see in the comment that I stubbed that out but they're not that close to the existing palette.  The shades are quite different and the design doesn't look as nice.